### ISSUE TITLE
MultipleNegativesSymmetricRankingLoss bug

### ISSUE THREAD

**Issue by mikescham** [UKPLab/sentence-transformers#1921]
!! 28/5/2023 - Nevermind, I was mistaken - transpose(0,1) is correct.  There is no bug.

I'm not sure if I'm correct but I believe the following line of code:

 backward_loss = self.cross_entropy_loss(**anchor_positive_scores.transpose(0, 1)**, labels)

should actually be using **anchor_positive_scores.transpose(1, 0)**

transpose(0,1) just returns the original matrix, right?  So if the forward loss didn't have any extra negative samples tagged on, then with the current code forward and backward loss would be equivalent.

**Comment by mikescham** [comment]
!! 28/5/2023 - Nevermind, I was mistaken - transpose(0,1) is correct. There is no bug.
